components/feral-watchdog/config.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sync"
+
+	"go.uber.org/zap"
+)
+
+const (
+	// Configuration file paths
+	DEBUG_CONFIG_FILE = "./feral-watchdog.json"
```

Comment:
I think we can remove it and assume using the same configuration for both modes.